README.md

```diff
@@ -277,6 +277,16 @@ You can gain access to the assets of your service as follows:
 
 Now, you have full access through the hot wallet to the EOAs addresses associated to your service and you can transfer their assets to any other address. You can also manage the assets of the service Safe through the DApp https://app.safe.global/, using the address located in the file `.trader_runner/service_safe_address.txt`.
 
+## Terminate your on-chain service
+
+If you wish to terminate your on-chain service (and receive back the staking/bonding funds to your owner/operator address in case your service is staked), ensure that your local service is stopped, and execute:
```

Comment:
Since you are suggesting to run the stop service script below:

```suggestion
If you wish to terminate your on-chain service (and receive back the staking/bonding funds to your owner/operator address in case your service is staked) execute:
```

Reply:
Why? I guess it's convenient not having the service running before modifying, so it's worth asking the user to do it.

Reply:
I think you missed the comment re the change.
We should either ask the user to stop the service and then run the terminate command or stop the service with the stop command and then terminate with the other command.

Reply:
ok, it's a very subtle issue. I commit your change